include/libdnf/rpm/solv_query.hpp

```diff
@@ -300,7 +300,7 @@ class SolvQuery {
 
     // TODO(jmracek) return std::pair<bool, std::unique_ptr<libdnf::rpm::Nevra>>
     /// @replaces libdnf/sack/query.hpp:method:std::pair<bool, std::unique_ptr<Nevra>> filterSubject(const char * subject, HyForm * forms, bool icase, bool with_nevra, bool with_provides, bool with_filenames);
-    std::pair<bool, libdnf::rpm::Nevra> subject_solution(const std::string & subject,
+    std::pair<bool, libdnf::rpm::Nevra> resolve_pkg_spec(const std::string & subject,
```

Comment:
It would be worth renaming the `subject` argument to `pkg_spec` as well.

Reply:
Good point